examples/involvingstates/example_pgm_pbm.py

```diff
@@ -0,0 +1,222 @@
+"""
+The Pretty Good and Pretty Bad Measurements
+===========================================
+
+In this tutorial, we will explore the "pretty good measurement" (PGM) and its
+novel counterpart, the "pretty bad measurement" (PBM), as introduced by
+McIrvin et.al :footcite:`McIrvin_2024_Pretty`. These measurements
+provide elegant, easy-to-construct tools for two opposing goals in quantum
+information: state discrimination and state exclusion.
+
+We will verify their core properties and replicate some of the key numerical
+results and figures from the paper using :code:`|toqito⟩`.
+"""
+
+# %%
+# Background: Discrimination vs. Exclusion
+# ----------------------------------------
+#
+# The standard quantum state discrimination task involves Alice sending Bob a
+# quantum state :math:`\rho_i` chosen from a known ensemble
+# :math:`\{(p_i, \rho_i)\}_{i=1}^k`. Bob's goal is to perform a measurement
+# that maximizes his probability of correctly guessing the index :math:`i`.
+# The best possible probability is denoted :math:`P_{\text{Best}}`. The "pretty good
+# measurement" (PGM) is a well-known and effective heuristic for this task.
+#
+# The state exclusion task is the opposite: Bob wins if he correctly guesses
+# a state that Alice *did not* send. This is equivalent to minimizing the
+# probability of correctly guessing the state Alice *did* send. The minimum
+# possible success probability for discrimination is :math:`P_{\text{Worst}}`. The
+# "pretty bad measurement" (PBM) is a new heuristic designed to approximate
+# this worst-case performance.
+#
+# A key result from :footcite:`McIrvin_2024_Pretty` is the tight relationship
+# between the success probabilities of these two measurements:
+#
+# .. math::
+#    P_{\text{PGM}} + (k-1)P_{\text{PBM}} = 1
+#
+# This implies a performance hierarchy against the optimal probabilities and the
+# blind guessing probability (:math:`1/k`):
+#
+# .. math::
+#    P_{\text{Best}} \ge P_{\text{PGM}} \ge \frac{1}{k} \ge P_{\text{PBM}} \ge P_{\text{Worst}}
+#
+# We will verify this hierarchy with a concrete example.
+
+# %%
+# Numerical Example: The Trine States
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# Figure 3 of the paper :footcite:`McIrvin_2024_Pretty` analyzes the performance
+# of these measurements for the three **trine states** with a uniform prior
+# probability. The trine states are a classic example of a set that is
+# antidistinguishable but not distinguishable.
```

Comment:
add a reference for this statement about the Trine state.  Either a paper or the quantum state exclusion tutorial in the docs that uses the trine example.